Gopkg.lock

```diff
@@ -145,17 +145,17 @@
   branch = "master"
   name = "github.com/prometheus/client_model"
   packages = ["go"]
-  revision = "5c3871d89910bfb32f5fcab2aa4b9ec68e65a99f"
+  revision = "fd36f4220a901265f90734c3183c5f0c91daa0b8"
```

Comment:
Was a new dep added? I don't see one; if not, don't update the deps yet, unless its necessary, but you can do it in another PR